SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialVirtualWorkerTranslator.cpp

```diff
@@ -23,27 +25,23 @@ void SpatialVirtualWorkerTranslator::Init(USpatialNetDriver* InNetDriver)
 	WorkerId = (NetDriver != nullptr) ? NetDriver->Connection->GetWorkerId() : "InvalidWorkerId";
 }
 
-void SpatialVirtualWorkerTranslator::SetDesiredVirtualWorkerCount(uint32 NumberOfVirtualWorkers)
+void SpatialVirtualWorkerTranslator::AddVirtualWorkerIds(const TSet<VirtualWorkerId>& InVirtualWorkerIds)
 {
 	// Currently, this should only be called once on startup. In the future we may allow for more
 	// flexibility. 
 	if (bIsReady)
 	{
 		UE_LOG(LogSpatialVirtualWorkerTranslator, Warning, TEXT("(%s) SetDesiredVirtualWorkerCount called after the translator is ready, ignoring."), *WorkerId);
+		return;
```

Comment:
Function name in above log should now be changed as well

Reply:
got it!